src/Table/Table.component.tsx

```diff
@@ -186,12 +186,13 @@ export class Table extends React.Component<Props> {
   };
 
   render() {
-    const { theme, getTrProps, ...props } = this.props;
+    const { data, theme, getTrProps, ...props } = this.props;
     return (
       <ThemeProvider theme={(outerTheme: any) => outerTheme || theme}>
         <SWrapperDiv
           minRows={0}
           pageSize={10}
+          showPagination={data.length > 0}
```

Comment:
What if the user want to set `showPagination` externally? 

I think it should be something like `showPagination={props.showPagination || (props.showPagination && data.length > 0)}`